modules/lofreq/1.0/lofreq.smk

```diff
@@ -129,10 +133,13 @@ rule _lofreq_filter_vcf:
         CFG["conda_envs"]["bcftools"]
     shell:
         op.as_one_line("""
-        bash {input.lofreq_filter} {input.vcf_all} | bgzip > {output.vcf_all_clean}
+        PATH={SCRIPT_PATH}:$PATH;
+        SCRIPT=$(which lofreq_filter.sh); 
+        echo $SCRIPT;
```

Comment:
Can you make this more informative? E.g. `echo "Using the script at $SCRIPT to filter lofreq calls"`

Reply:
Done but not tested. Do you happen to know if quotes necessary?

Reply:
I would use quotes to be safe, yes, to make sure each word in the echo phrase isn't parsed as a separate argument.